slider.css

```diff
@@ -0,0 +1,57 @@
+.slider {
+  display: block;
+}
+
+.slider-container {
+  display: inline-block;
+  position: relative;
+}
+
+.slider-list {
+  padding: 0;
+  margin: 0;
+}
+
+.slider-item {
+  display: none;
+}
+
+.slider-item-active {
+  display: block;
+}
+
+.navigation-buttons {
+  display: none;
+}
+
+.navigation-buttons .btn {
+  position: absolute;
+  top: 50%;
+  transform: translate(0, -50%);
+}
+
+.navigation-buttons .btn.btn-prev {
+  left: 0;
+}
+
+.navigation-buttons .btn.btn-next {
+  right: 0;
+}
+
+.showNavigationButtons {
+  display: block;
+}
+
+.placeholder {
+  width: 400px;
+  height: 200px;
+  display: none;
+}
+
+.placeholder.show-placeholder {
```

Comment:
something is wrong with this placeholder. Unfortunately, the logic looks not straightforward and it's difficult to understand how it works without running a project.

Reply:
I don't understand. Would it be better to remove placeholder from HTML and add it dynamically when slides.length === 0 ?